invenio_banners/services/results.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Banners is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Service results."""
+from invenio_records_resources.services.records.results import RecordItem, \
+    RecordList
+
+
+class BannerItem(RecordItem):
+    """Single banner result."""
+
+    def __init__(
+        self,
+        service,
+        identity,
+        banner,
+        links_tpl=None,
+        errors=None,
+        schema=None,
+    ):
+        """Constructor."""
+        self._banner = banner
+        self._identity = identity
+        self._service = service
+        self._data = None
+        self._schema = schema or service.schema
+        self._links_tpl = links_tpl
+        self._errors = errors
+
+    @property
+    def errors(self):
```

Comment:
I think that you can omit to re-implement some methods, which are the same in the base class?
This helps to keep this class smaller and more readable :)